src/index.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Header from './Header';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
+    <Header />
```

Comment:
we should move this into the `App` component. It doesn't look like we're going to put any user specific information into the header, so we can just leave it in the `App`

Reply:
I'd like to keep Header as its own component, currently that's not necessary, but it may be in the future, plus small re-usable components seems like a good idea.

Reply:
Ah, sorry if I wasn't clear. I didn't mean to get rid of  the header component. I meant to put the header component in the App component instead of this index file.

Reply:
if "It doesn't look like we're going to put any user specific information into the header, so we can just leave it in the App" caused confusion, I meant that we can call the header component directly from the App component, instead of creating something like a `PageLayout` component that contains the header and other page layout information that we call from each individual page component

Reply:
Thanks for the clarification, reading is hard sometimes for me lol... you're definitely right, it has been moved.